### ISSUE TITLE
Question about domain division in DDLearn

### ISSUE THREAD

**Issue by ehappymonkey** [microsoft/robustlearn#65]
In the file data_preprocess_devide_domain.py file line 126-130, I believe the authors conclude every domain knowledge into the training set, instead of treating a separate domain as the testing set. This could lead to the collapse of the whole OOD setting. Please correct me if I am wrong. The code :  
        x_i, y_i = data_lst[i][0], data_lst[i][1]
        x_train_i, x_tmp_i, y_train_i, y_tmp_i = train_test_split(x_i, y_i, test_size=0.4, random_state=seed, stratify=y_i)
        x_val_i, x_test_i, y_val_i, y_test_i = train_test_split(x_tmp_i, y_tmp_i, test_size=0.5, random_state=seed, stratify=y_tmp_i)

**Comment by ehappymonkey** [comment]
The authors do provided the right-splitted DSADS dataset. I would appreciate to a lot if you can upload the splitting code or/and the other two splitter datasets!

**Comment by ehappymonkey** [comment]
The authors do provided the right-splitted DSADS dataset. I would appreciate it a lot if you can upload the splitting code or/and the other two splitter datasets!
